docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,68 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |filter contacts by name, number, address, email or tag |find specific contacts easily
+
+|`* * *` |basic user |add display pictures to my contacts |easily identify people with the same name
+
+|`* * *` |basic user |call/email my contacts by clicking on them |save time in opening my email or phoning app
+
+|`* * *` |basic user |be able to change the theme of the app from dark to light |reduce the stress on my eyes
+
+|`* * *` |basic user |customise the background picture of the app |enhance my user experience since it looks nicer
```

Comment:
I would argue this user story and the one above are more of `*`-priority features